tests/test_bonfire.py

```diff
@@ -0,0 +1,148 @@
+import pytest
+from click.testing import CliRunner
+import json
+
+from bonfire import bonfire
+
+
+namespace_data = {}
+reservation_data = {}
+
+with open("data/namespace_data.json", 'r') as namespace_data_file:
```

Comment:
Let's load these files in a module-scoped fixture instead.

Also let's define a data path dynamically... if I run `pytest .` in the root dir, the tests are unable to locate `data/namespace_data.json`

Something along these lines should work:
```python
from pathlib import Path

DATA_PATH = Path(__file__).parent.joinpath("data")

@pytest.fixture(scope="module")
def namespace_list():
    with open(DATA_PATH.joinpath("namespace_data.json")) as namespace_data_file:
        return json.load(namespace_data_file)["items"]
```